aten/src/ATen/native/cuda/Dropout.cu

```diff
@@ -107,10 +116,14 @@ fused_dropout_kernel_vec(at::cuda::detail::TensorInfo<const scalar_t, IndexType>
 
     // Perform the actual computation
     #pragma unroll
-    for (int ii = 0; ii < VEC; ii++) {
-      r[ii] = src[ii]*(&rand.x)[ii]*scale;
-      mask[ii] = (mask_t)(&rand.x)[ii];
+    for (int jj = 0; jj < RAND_SIZE; jj++) {
+      #pragma unroll
```

Comment:
Outer loop will one run ones for non-ROCM.
Inner loop is basically same as old code.